controllers/azuremachine_controller_test.go

```diff
@@ -141,7 +143,7 @@ func TestAzureMachineReconcile(t *testing.T) {
 
 			resultIdentity := &infrav1.AzureClusterIdentity{}
 			key := client.ObjectKey{Name: defaultAzureClusterIdentity.Name, Namespace: defaultAzureClusterIdentity.Namespace}
-			g.Expect(fakeClient.Get(context.TODO(), key, resultIdentity))
+			g.Expect(fakeClient.Get(context.TODO(), key, resultIdentity)).To(Succeed())
```

Comment:
The `ginkgo-linter` catches these no-op expressions and requires an assertion.

In this case, that uncovered a bug: `fakeClient.Get()` was actually returning an error about "can't find fake-identity" for the last two test cases.